crates/miden-objects/src/account/component/mod.rs

```diff
@@ -146,6 +146,18 @@ impl AccountComponent {
         self.supported_types.contains(&account_type)
     }
 
+    /// Returns a vector of tuples (digest, is_auth) for all procedures in this component.
+    pub fn get_procedures(&self) -> Vec<(Digest, bool)> {
```

Comment:
Not sure if this is generally useful enough to be `pub` due to returning the `is_auth` flag, so I'd make this `pub(super)` or `pub(crate)`.

Reply:
Sure, done in [change visibility of get_procedures](https://github.com/0xMiden/miden-base/pull/1480/commits/418b8455356d431a8db4626583cd91c8b71c6fd8)